front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/InputsTable.test.tsx

```diff
@@ -150,6 +177,173 @@ describe('Insights Input Table', () => {
       mockInputData.list[0].relationships.categories.data[0].id
     );
   });
+
+  // Rows selection and actions
```

Comment:
Let's add these types of comments to describe blocks? I think it will make it clearer with the actions, also with the pagination maybe?

Reply:
I've tried to name the code blocks in the index file, hope that's what you meant. Super hard to do so, tough, with effects and callbacks that can't go after the null retrun for the hooks ordering, and the other functions tht rely on the data being there.

Reply:
Sorry, I should have been more clear. All I meant was to group related tests in 'describe' blocks nested inside the general describe block just to make it look tidier in the test file. Like `describe('Actions', ()=>{ ... tests ...})`, etc.
Moving things around in the index file would indeed be difficult and is also not super necessary, imo :)

Reply:
Ah, right, done.